sources/packages/backend/apps/api/src/services/student-file/student-file.service.ts

```diff
@@ -48,20 +49,23 @@ export class StudentFileService extends RecordDataModelService<StudentFile> {
     newFile.fileName = createFile.fileName;
     newFile.uniqueFileName = createFile.uniqueFileName;
     newFile.groupName = createFile.groupName;
-    newFile.mimeType = createFile.mimeType;
-    newFile.fileContent = createFile.fileContent;
     newFile.student = { id: studentId } as Student;
     newFile.creator = { id: auditUserId } as User;
     newFile.virusScanStatus = VirusScanStatus.InProgress;
 
     const summary = new ProcessSummary();
     let savedFile: StudentFile;
-    // Save the file to db.
     try {
       savedFile = await this.repo.save(newFile);
     } catch (error: unknown) {
       this.logger.error(`Error saving the file: ${error}`);
-      return;
+      this.logger.error(
+        `Uploading file ${newFile.fileName} to S3 storage failed. Error persisting the file details to the database.`,
```

Comment:
The error messages should be related to the operation. There is no S3 storage operation happening here.
The below should be enough.
```ts
Error persisting the file details to the database.
```